tests/test_pyrflx.py

```diff
@@ -14,27 +17,35 @@
 )
 from rflx.pyrflx import (
     EnumValue,
-    Field,
     IntegerValue,
-    Message,
+    MessageValue,
     NotInitializedError,
     OpaqueValue,
     Package,
     PyRFLX,
     TypeValue,
 )
+# pylint: disable=too-many-public-methods
+# pylint: disable=too-many-instance-attributes
+# pylint: disable=too-many-lines
```

Comment:
Please move that to the to be affected entity, I guess this is `TestPyRFLX` in this case. You can also merge that into one line: `# pylint: disable=too-many-public-methods,too-many-instance-attributes,too-many-lines`.

If you have the rare case that you want to disable a check for the whole file, move the comment to the first line of the file.